ArcGISRuntimeSDKQt_QMLSamples/Maps/MapReferenceScale/MapReferenceScale.qml

```diff
@@ -0,0 +1,253 @@
+    clip: true
+    width: 800
+    height: 600
+
+    property var referenceScales: [500000,250000,100000,50000]
+    property string webMapId: "3953413f3bd34e53a42bf70f2937a408"
+    property LayerListModel featureLayerModel
+
+    MapView{
+        id: mapView
+        anchors.fill: parent
+
+        // Create a Map from a Portal Item
+        Map {
+            id: map
+
+            PortalItem {
+                id: mapPortalItem
+                portal: portal
+                itemId: webMapId
+                onErrorChanged: {
+                        console.log("Error in portal item: " + error.message + "(" + error.additionalMessage + ")");
+                }
+                onLoadStatusChanged: {
+                        if (loadStatus === Enums.LoadStatusLoaded) {
+                            //load operational layers into listmodel
+                            featureLayerModel = map.operationalLayers
+                        }
+                }
+            }
+        }
+    }
+
+    Rectangle {
+        id: referenceScaleRect
+        anchors {
+            margins: 10
+            left: parent.left
+            top: parent.top
+        }
+        height: 150
+        width: 200
+        color: "transparent"
+
+        MouseArea {
+            anchors.fill: parent
+            onClicked: mouse.accepted = true
+            onWheel: wheel.accepted = true
+        }
+
+        Rectangle {
+            anchors.fill: parent
+            width: referenceScaleRect.width
+            height: referenceScaleRect.height
+            color: "#000000"
+            opacity: .75
+            radius: 5
+            border {
+                color: "#4D4D4D"
+                width: 1
+            }
+
+            ColumnLayout {
+                id: referenceScaleLayout
+                anchors {
+                    fill: parent
+                    margins: 5
+                }
+
+                Text {
+                    color: "#ffffff"
+                    text: qsTr("Current Map Scale 1:" + Math.round(mapView.mapScale))
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
+                    verticalAlignment: Text.AlignBottom
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                    }
+                    clip: true
+                }
+
+                Text {
+                    color: "#ffffff"
+                    text: qsTr("Select a new reference scale")
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
+                    verticalAlignment: Text.AlignBottom
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
+                    }
+                    clip: true
+                }
+
+                ComboBox {
+                    id: scales
+                    font {
+                        weight: Font.DemiBold
+                        pixelSize: 11
```

Comment:
There's an issue with pixelSize used across devices being inconsistent. Consider `pointSize` instead. Goes for all instances of `pixelSize`.